distribution/src/bin-regular/hz-start.bat

```diff
@@ -41,7 +41,16 @@ IF NOT "%JAVA_VERSION%" == "8" (
 
 REM HAZELCAST_CONFIG holds path to the configuration file. The path is relative to the Hazelcast installation (HAZELCAST_HOME).
 if "x%HAZELCAST_CONFIG%" == "x" (
-    set HAZELCAST_CONFIG=config/hazelcast.xml
```

Comment:
I think we want to keep this line.

Reply:
Yes, that was a mistake.

Reply:
I changed the slash though, so it still shows as deleted, but it' added below.